### PR TITLE
Add two Sourcegraph videos

### PR DESCRIPTION
**Pull request by ltagliaferri**
### What should this PR do?
Adds two videos, resolving DEVED-130 and DEVED-131.

### Why are we making this change?
Product-related educational video resources.

### What are the acceptance criteria? 
Ensure that embedded YouTube videos are correct and play. Ensure that tutorials are formatted correctly. 

### How should this PR be tested?
Check out branch locally and check the frontend of the site, or else check the deployment preview. 

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.

### CODE REVIEW

File: posts/three-ways-to-search-code-with-sourcegraph.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Three ways to search code with Sourcegraph
 tags: [video, search, sourcegraph]
 author: marek-zaluski
+description: Demo of how to search code three different ways with Sourcegraph
```
Comment (ltagliaferri) [inline posts/three-ways-to-search-code-with-sourcegraph.md:5]: I noticed this description was missing when previewing this locally

File: components/layouts/Layout.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ const SITE_TITLE = 'Sourcegraph Learn'
 
 const defaultMetaTags = {
     description: 'Sourcegraph Learn is an educational hub to support all developers.',
-    image: '/headers/sourcegraph-learn-header.png',
+    image: 'https://storage.googleapis.com/sourcegraph-assets/learn/headers/sourcegraph-learn-header.png',
```
Comment (ltagliaferri) [inline components/layouts/Layout.tsx:12]: I missed this in #178

File: posts/how-to-install-sourcegraph-on-linux-with-docker.md [hunk 3]
```diff
@@ -25,15 +25,15 @@ On a Linux cloud server or local computer, you can install Docker via the comman
 
 Ensure that you have the [snap package manager](https://snapcraft.io/) installed by running the following commands in your terminal for **Ubuntu or Debian Linux**:
 
-```
+```bash
```
Comment (ltagliaferri) [inline posts/how-to-install-sourcegraph-on-linux-with-docker.md:28]: I noticed I hadn't made rendered these as bash the first time around, fixing now

Comment (katjuell) [reply posts/how-to-install-sourcegraph-on-linux-with-docker.md:28]: I notice we use `sh` in the README (looks like that convention was first introduced [here](https://github.com/sourcegraph/learn/commit/12684d8be7a8bdee3b340dc002cc568966621f63)). Personally, I don't care what we use as our "default", but I guess we should stay consistent. Do you have any preferences?

Comment (ltagliaferri) [reply posts/how-to-install-sourcegraph-on-linux-with-docker.md:28]: Ah, I can update to `sh`

File: posts/video-how-to-add-open-source-software-to-sourcegraph-cloud.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: Video — How to add open source software to Sourcegraph cloud
+author: lisa-tagliaferri
+tags: [video, Sourcegraph, open source]
+description: Find out how to add open source repos to Sourcegraph cloud so you can search across all the code you care about.
+image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/install-sourcegraph-with-docker-screengrab.png
```
Comment (katjuell) [inline posts/video-how-to-add-open-source-software-to-sourcegraph-cloud.md:6]: Do you have thoughts on how you want to handle longer nested headers in ToC? 
Screen Shot 2021-08-30 at 3 10 52 PM

Here's a long-ish primary header, for context:
Screen Shot 2021-08-30 at 3 10 41 PM

Happy to address whatever you want to do in another PR that I can then merge into this.

Comment (ltagliaferri) [reply posts/video-how-to-add-open-source-software-to-sourcegraph-cloud.md:6]: Did the wrap change? On local and deploy I am getting this 

Screen Shot 2021-08-30 at 4 00 24 PM

Comment (katjuell) [reply posts/video-how-to-add-open-source-software-to-sourcegraph-cloud.md:6]: For screens above `1464` — so monitors etc. 
I've got a PR to merge into this branch: https://github.com/sourcegraph/learn/pull/188

**Review by katjuell — APPROVED** [review]
lgtm!

**Pull request merged**